### PR TITLE
Python 3 support 

### PR DESCRIPTION
**Pull request by givanfp**
- This adds python 3 support for dronekit-sitl, needed for dronekit-python and python 3 compatibility
- Added dependencies: 'Six'
- Created new test: 'test_version_list'

### CODE REVIEW

**Comment by hamishwillee** [comment]
@givanfp Thanks very much. Very cool! 

@tcr3dr you will have to verify this. 

I believe that we need to use pip2 to install Python3 or Python 2 binaries right?
What else needs to be done? Minimally updating the examples to run on Python 3?

File: dronekit_sitl/__init__.py [hunk 1]
```diff
@@ -103,7 +117,14 @@ def download(system, version, target, verbose=False):
         if not os.path.isdir(sitl_target):
             os.makedirs(sitl_target)
 
-        testfile = urllib.URLopener()
+        if six.PY3:
```
Comment (mrpollo) [inline dronekit_sitl/__init__.py]: I don't have experience working with `six` but from the docs seems like using `six.moves.urllib` would be a better approach here, what do you think?

File: dronekit_sitl/__init__.py [hunk 2]
```diff
@@ -10,8 +11,8 @@
 import json
 import tarfile
 import sys
-import urllib
-import urllib2
+#import urllib
+#import urllib2
```
Comment (mrpollo) [inline dronekit_sitl/__init__.py]: can we just remove instead of commenting them?

File: dronekit_sitl/__init__.py (hunk 1 above)
Comment (givanfp) [inline dronekit_sitl/__init__.py]: You're right. It's a more clean solution.

**Comment by mrpollo** [comment]
wonder if we should update either circleci or travis to run under python 3 before merging, what do you guys think @tcr3dr @hamishwillee ?

**Comment by hamishwillee** [comment]
@mrpollo I don't know the implications of (not) doing this. If we support Python 3 we need to know that the library runs on both Python 2 and Python 3, so if that is what is required to ensure changes on both versions are fully tested, then yes :-)

**Comment by mrpollo** [comment]
that's correct @hamishwillee, we can't test if python3 works unless we get a test environment

**Comment by mrpollo** [comment]
just submitted #66 which should be merged in together with this PR

**Comment by mrpollo** [comment]
merging

**Comment by tcr3dr** [comment]
Thank you @givanfp :)

**Pull request merged**